README.md

```diff
@@ -20,10 +22,37 @@ range of applications using the [Google Maps Android API][android-site].
 
 <p align="center"><img width="90%" vspace="20" src="https://cloud.githubusercontent.com/assets/1950036/6629704/f57bc6d8-c908-11e4-815a-0d909fe02f99.gif"></p>
 
+## Requirements
+
+* Android API level 15+
+* Maps SDK via Google Play Services (this library is not yet compatible with the [standalone Maps SDK BETA library])
```

Comment:
I believe this is broadly referred to as "Maps SDK v3.0 BETA", so I'd suggest adding the "v3.0" to make it clearer what this is referring to.